regression/cbmc/string-refinement-strchr2/test.c

```diff
@@ -0,0 +1,13 @@
+#include <assert.h>
+#include <string.h>
+
+int main()
+{
+  int i;
+  char *test = i ? "fo\0tis" : "notfoti";
```

Comment:
There should be a test for the case when a character can’t be found as well, e.g. in this case `'t'` may or may not be found depending on `i`.